python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -203,7 +203,13 @@ def _node_refreshed(self, item):
                 "Model item refreshed: %s", item.data(self.parent()._SG_ITEM_UNIQUE_ID)
             )
             self.parent()._node_refreshed.disconnect(self._node_refreshed)
-            self.deleteLater()
+            # We've seen problems in Qt5 with deleteLater causing
```

Comment:
Same comment as the qtwidgets CR. I think if we're going to do this, we should do it via method in `sgtk.platform.qt`. What do you think?